Source/Core/VideoCommon/VertexShaderManager.cpp

```diff
@@ -437,15 +440,28 @@ void VertexShaderManager::SetConstants()
 			g_fProjectionMatrix[15] = 0.0f;
 
 			// Heuristic to detect if a GameCube game is in 16:9 anamorphic widescreen mode.
+			float temp_ratio, vp_ratio;
+			temp_ratio = fabsf(rawProjection[2] / rawProjection[0]);
```

Comment:
doing this gives me compilation errors, and I'm not sure why?

It says "switch case is in protected scope" and "jump bypasses variable initialization".  If I declare the variables on separate lines, then all of the errors go away.  Any insight?